tests/test_bids.py

```diff
@@ -161,52 +182,45 @@ def test_update_bids_only_updates_selected_fields(self):
         response = self.test_app.put(
             '/bids/1',
             json={
-                'bidder_name': 'Updated Name',
-                'bidder_email': 'new@email.com'
+                'user_id': 3,
+                'amount': 999.15
             },
             follow_redirects=True
         )
 
         self.assertEquals(response.status, "200 OK")
         payload = json.loads(response.data)
-        self.assertEquals(payload['bidder_name'], 'Updated Name')
-        self.assertEquals(payload['bidder_email'], 'new@email.com')
-        self.assertEquals(payload['zip_code'], '12345')
-        self.assertEquals(payload['city'], 'My City')
+        self.assertEquals(payload['user_id'], 3)
+        self.assertEquals(payload['amount'], 999.15)
+        self.assertEquals(payload['item_id'], 1)
+        self.assertEquals(payload['winner'], False)
 
     def test_update_can_update_all_fields(self):
         response = self.test_app.put(
             '/bids/1',
             json={
-                'bidder_name': 'Joe Strummer',
-                'bidder_email': 'clampdown@clash.com',
-                'amount': 19.79,
-                'street_address': '123 Clash St.',
-                'city': 'Hitsville',
-                'state': 'UK',
-                'zip_code': '99999',
-                'receipt': 'www.death_or_glory.com'
+                'item_id': 2,
+                'user_id': 3,
+                'amount': 500.00,
+                'winner': True
             },
             follow_redirects=True
         )
 
         self.assertEquals(response.status, "200 OK")
         payload = json.loads(response.data)
-        self.assertEquals(payload['bidder_name'], 'Joe Strummer')
-        self.assertEquals(payload['bidder_email'], 'clampdown@clash.com')
-        self.assertEquals(payload['amount'], 19.79)
-        self.assertEquals(payload['street_address'], '123 Clash St.')
-        self.assertEquals(payload['city'], 'Hitsville')
-        self.assertEquals(payload['state'], 'UK')
-        self.assertEquals(payload['zip_code'], '99999')
-        self.assertEquals(payload['receipt'], 'www.death_or_glory.com')
+        self.assertEquals(payload['item_id'], 2)
+        self.assertEquals(payload['user_id'], 3)
+        self.assertEquals(payload['amount'], 500.00)
+        self.assertEquals(payload['winner'], True)
 
     def test_sad_path_for_update_bid(self):
         response = self.test_app.put(
             '/bids/1111',
             json={
-                'bidder_name': 'Joe Strummer',
-                'bidder_email': 'clampdown@clash.com'
+                'item_id': 2,
+                'user_id': 3,
+                'amount': 500.00
             },
```

Comment:
So much little nitty gritty stuff in the tests! Great job taking the time to go through all that!